Blockzilla/OverlayView.swift

```diff
@@ -197,20 +186,20 @@ class OverlayView: UIView {
             } else if findInPageButton.isHidden {
                 copyButton.snp.remakeConstraints { make in
                     make.leading.trailing.equalTo(safeAreaLayoutGuide)
-                    make.top.equalTo(searchBorder)
+                    make.top.equalTo(searchButton.snp.bottom)
                     make.height.equalTo(56)
                 }
             } else {
                 copyButton.snp.remakeConstraints { make in
                     make.leading.trailing.equalTo(safeAreaLayoutGuide)
-                    make.top.equalTo(findInPageBorder)
+                    make.top.equalTo(findInPageButton.snp.bottom)
                     make.height.equalTo(56)
                 }
             }
         } else {
             copyButton.isHidden = true
-            copyBorder.isHidden = true
         }
+        layoutIfNeeded()
```

Comment:
nit: confirm this call is truly needed. Ideally the system does this for us.

Reply:
Confirmed needed :)